robotbase/src/main/java/com/team2357/frc2022/Constants.java

```diff
@@ -147,6 +147,13 @@
         public static final String ARDUINO_SENSOR_DEVICE_NAME = "/dev/ttyACM0";
     }
 
+    public final class SENSORS {
+        public static final int FEEDER_SENSOR_DIO_PORT = -1;
```

Comment:
This will probably break the robot code, just put ```0``` for intake and ```1``` for feeder for now.